packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -17,7 +17,7 @@
 
 log = logging.getLogger(__name__)
 
-GLOBAL_PORT_POOL = collections.defaultdict(lambda: list(range(10000, 32000)))
+GLOBAL_PORT_POOL = collections.defaultdict(lambda: list(range(10000, 14849)) + list(range(15104, 32000)))
```

Comment:
Can we please have here a link to a Jira comment or a short description of the problem?

This way we will avoid "magic numbers" in the code and people maintaining this code, later on, will not be tempted to remove it until we bump CentOS image we use.

Reply:
+1